File: 3153/3153.go
func sumOfDigitDifferencesOfAllPair(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := sumOfDigitDifferencesOfAllPair(root.Left)
	r := sumOfDigitDifferencesOfAllPair(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 3153/3153.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var sumOfDigitDifferencesOfAllPair = function(root) {
    if (!root) return 0;
    return 1 + Math.max(sumOfDigitDifferencesOfAllPair(root.left), sumOfDigitDifferencesOfAllPair(root.right));
};